arb_os/decompression.mini

```diff
@@ -342,3 +328,7 @@ impure func decompressTx(
     ));
 }
 
+func dumbHack() {
```

Comment:
Add a comment explaining why this is here.

Reply:
Added.